Sources/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsPredicate.swift

```diff
@@ -35,3 +35,16 @@ public protocol UIKitRUMViewsPredicate {
     /// - Returns: RUM View parameters if received view controller should start/end the RUM View, `nil` otherwise.
     func rumView(for viewController: UIViewController) -> RUMView?
 }
+
+/// Default implementation of `UIKitRUMViewsPredicate`.
+/// It names  RUM Views by the names of their `UIViewController` subclasses.
+public struct DefaultUIKitRUMViewsPredicate: UIKitRUMViewsPredicate {
+    public init () {}
+
+    public func rumView(for viewController: UIViewController) -> RUMView? {
+        let className = NSStringFromClass(type(of: viewController))
+        let isCustomClass = className.contains(".") // custom class contains module prefix
+
+        return isCustomClass ? RUMView(path: className) : nil
```

Comment:
what does custom class mean? most of the view controllers will have `module_name.class_name` as their name, won't they?

we can use `Mirror` here instead of returning `nil`

Reply:
Custom means not a vanilla `UIKit` class (e.g. not the `UIViewController`). This is further explained in the test:
```swift
func testGivenDefaultPredicate_whenAskingUIKitViewController_itReturnsNoView()
```